### ISSUE TITLE
ci-gce-kubeadm-1.11 tests failing in release-1.11

### ISSUE THREAD

**Issue by foxish** [kubernetes/kubernetes#66250]
ci-gce-kubeadm-1.11 tests are currently failing and blocking the cut of 1.11.

```
I0712 11:45:33.486] Trying to fetch kubeconfig from master... 59/60
W0712 11:45:33.587] + for tries in '{1..60}'
W0712 11:45:33.587] + echo Trying to fetch kubeconfig from master... 59/60
W0712 11:45:33.587] + gcloud compute ssh --project k8s-jkns-e2e-gce --zone us-central1-f prow@e2e-165-471dd-master --command 'echo STARTFILE; sudo cat /etc/kubernetes/admin.conf'
W0712 11:45:34.639] cat: /etc/kubernetes/admin.conf: No such file or directory
W0712 11:45:34.736] + sleep 5
W0712 11:45:39.738] + for tries in '{1..60}'
W0712 11:45:39.738] + echo Trying to fetch kubeconfig from master... 60/60
W0712 11:45:39.738] + gcloud compute ssh --project k8s-jkns-e2e-gce --zone us-central1-f prow@e2e-165-471dd-master --command 'echo STARTFILE; sudo cat /etc/kubernetes/admin.conf'
I0712 11:45:39.839] Trying to fetch kubeconfig from master... 60/60
W0712 11:45:40.852] cat: /etc/kubernetes/admin.conf: No such file or directory
W0712 11:45:40.957] + sleep 5
W0712 11:45:45.959] + echo Exhausted attempts to fetch kubeconfig.
W0712 11:45:45.960] Exhausted attempts to fetch kubeconfig.
W0712 11:45:45.960] + exit 1
W0712 11:45:45.960] make[1]: *** [do] Error 1
W0712 11:45:45.961] make: *** [deploy-cluster] Error 2
W0712 11:45:45.962] 2018/07/12 11:45:45 process.go:155: Step 'make -C /workspace/kubernetes-anywhere WAIT_FOR_KUBECONFIG=y deploy' finished in 7m2.884297899s
```

Ref: http://k8s-testgrid.appspot.com/sig-release-1.11-blocking#ci-gce-kubeadm-1.11

cc/ @timothysc @luxas

**Comment by foxish** [comment]
cc/ @kubernetes/sig-cluster-lifecycle-bugs

**Comment by timothysc** [comment]
>Jul 12 11:40:44 e2e-165-471dd-master startup-script: INFO startup-script: + kubeadm init --skip-preflight-checks --config /etc/kubeadm/kubeadm.yaml
Jul 12 11:40:44 e2e-165-471dd-master startup-script: INFO startup-script: /startup-y1g0u54q/tmp0_6vup9v: line 211: kubeadm: command not found
Jul 12 11:40:44 e2e-165-471dd-master startup-script: INFO startup-script: Return code 127. 

Something changed on the deployment w/test-infra.  I don't see any PRs that would affect this way.  Initial error doesn't look like us.  
 
/cc @kubernetes/test-infra-maintainers @BenTheElder @luxas

**Comment by BenTheElder** [comment]
Nothing changed infra side between the last success the failures https://github.com/kubernetes/test-infra/compare/6050551de...fbe6fd779

We also didn't deploy anything on 07/11 or 07/12 (see #testing-ops on slack)

Almost all of the kubeadm deployment logic is in kubernetes-anywhere. I have no idea if anything changed there.

**Comment by timothysc** [comment]
Did something change with the release tooling?  

Here is the link to the log above - https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-1-11/165/artifacts/e2e-165-471dd-master/serial-1.log

**Comment by BenTheElder** [comment]
a few things changed with those, mostly related to kubeadm debs for actual releases https://github.com/kubernetes/release/commits/master

**Comment by timothysc** [comment]
The test failures are caused by this cherry-pick - https://github.com/kubernetes/kubernetes/pull/65921

**Comment by timothysc** [comment]
```
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script: dpkg: dependency problems prevent configuration of kubeadm:
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script:  kubeadm depends on kubelet (>= 1.8.0); however:
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script:   Package kubelet is not configured yet.
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script:  kubeadm depends on cri-tools (>= 1.11.0); however:
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script:   Package cri-tools is not installed.
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script: dpkg: error processing package kubeadm (--install):
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script:  dependency problems - leaving unconfigured
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script: Setting up kubectl (1.11.1~beta.0.83+e3db7214997c40) ...
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script: Setting up kubernetes-cni (0.5.1) ...
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script: Errors were encountered while processing:
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script:  kubelet
Jul 12 11:40:39 e2e-165-471dd-master startup-script: INFO startup-script:  kubeadm
```

Looks like it's failing to find cri-tools 1.11.+... It's in the release repo, but it's consuming bazel artifacts in the tests.  
@luxas @Random-Liu @chuckha

**Comment by timothysc** [comment]
We need to fix the builds for 1.11 to update the cri-tools. 

/cc @feiskyer @Random-Liu

**Comment by luxas** [comment]
With this stuff should be fixed: https://github.com/kubernetes/test-infra/pull/8697
